src/l500/l500-color.h

```diff
@@ -129,6 +130,12 @@ namespace librealsense
        
         // Stops the color sensor if was opened by the calibration process, otherwise does nothing
         void stop_stream_for_calibration();
+
+        void init_calibration_controls()
```

Comment:
Please take function bodies out of the header

Reply:
This is called only once, right? In that case, why have it as a separate function? It's very simple...